portfolio/src/main/webapp/script.js

```diff
@@ -73,11 +73,11 @@ function contact() {
 
 function getComments() {
 	fetch('/data').then(response => response.json()).then((comments) => {
-		console.log(comments);
 		const bodyElement = document.body;
-		bodyElement.innerHTML += 'Here are the comments: \n';
-        for (comment of comments) {
-            bodyElement.innerHTML += comment + '\n';
+        console.log('Comments: ', comments);
+        bodyElement.innerHTML += 'Here are all the comments: <br>';
+        for (index in comments) {
+    		bodyElement.innerHTML += comments[index] + '<br>';
```

Comment:
nit - These line look under-indented in this block.

Reply:
hmm I don't see anything odd on my end :(

Reply:
Line 77 (and on) is indented less than 76:

![image](https://user-images.githubusercontent.com/2066940/86948128-9361a700-c101-11ea-8755-0f942e28f726.png)